### ISSUE TITLE
Undefined symbol "__gxx_personality_v0"

### ISSUE THREAD

**Issue by eserte** [vovkasm/data-binarybuffer#1]
On my freebsd smokers (freebsd 11 - 13) the test suite fails:
```
#   Failed test 'Data/BinaryBuffer.pm loaded ok'
#   at t/00-compile.t line 31.
#          got: '65280'
#     expected: '0'
Can't load '/usr/home/cpansand/.cpan/build/2019043005/Data-BinaryBuffer-0.005-1/blib/arch/auto/Data/BinaryBuffer/BinaryBuffer.so' for module Data::BinaryBuffer: /usr/home/cpansand/.cpan/build/2019043005/Data-BinaryBuffer-0.005-1/blib/arch/auto/Data/BinaryBuffer/BinaryBuffer.so: Undefined symbol "__gxx_personality_v0" at /usr/local/lib/perl5/5.28/mach/DynaLoader.pm line 193.
Compilation failed in require at -e line 1.
# Looks like you failed 1 test of 1.
t/00-compile.t ......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

Note that freebsd has clang as default compiler, and it seems that a gcc-specific symbol is used here.

**Comment by vovkasm** [comment]
Thanks! Will check today

**Comment by vovkasm** [comment]
Should be fixed in v0.006. Will wait cpantesters result before closing.

**Comment by vovkasm** [comment]
Closing, now situation with testers definitily better - http://matrix.cpantesters.org/?dist=Data-BinaryBuffer+0.006
At least this error fixed.
